plugins/performance-lab/includes/site-health/avif-headers/helper.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Helper function to detect and test AVIF header information.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for avif_headers test.
+ *
+ * @since n.e.x.t
+ *
+ * @return array{label: string, status: string, badge: array{label: string, color: string}, description: string, actions: string, test: string} Result.
+ */
+function avif_headers_check_avif_headers_test(): array {
+	$result = array(
+		'label'       => __( 'Your site sends AVIF image headers', 'performance-lab' ),
```

Comment:
I can't remember the specifics of escaping fields like this.

Reply:
I will match other instances

Reply:
Escaping for labels is only included in `plugins/performance-lab/tests/data/class-site-health-mock-responses.php`, maybe we should remove it there.